specs/_features/eip7547/p2p-networking.md

```diff
@@ -0,0 +1,93 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [EIP-7547 -- Networking](#eip-7547----networking)
+  - [Preset](#preset)
+    - [Execution](#execution)
+  - [Containers](#containers)
+    - [New Containers](#new-containers)
+      - [`SignedInclusionList`](#signedinclusionlist)
+      - [`SignedBeaconBlockAndInclusionList`](#signedbeaconblockandinclusionlist)
+  - [Modifications in EIP7547](#modifications-in-eip7547)
+    - [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+      - [Topics and messages](#topics-and-messages)
+        - [Global topics](#global-topics)
+          - [`beacon_block`](#beacon_block)
+      - [Transitioning the gossip](#transitioning-the-gossip)
+  - [Design rationale](#design-rationale)
+  - [Why is it proposer may send multiple inclusion lists? Why not just one per slot?](#why-is-it-proposer-may-send-multiple-inclusion-lists-why-not-just-one-per-slot)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# EIP-7547 -- Networking
+
+This document contains the consensus-layer networking specification for EIP-7547.
+
+The specification of these changes continues in the same format as the network specifications of previous upgrades, and assumes them as pre-requisite.
+
+## Preset
+
+### Execution
+
+| Name | Value |
+| - | - |
+| `MAX_TRANSACTIONS_PER_INCLUSION_LIST` |  `uint64(143)` |
+
+## Containers
+
+### New Containers
+
+#### `SignedInclusionList`
+
+```python
+class SignedInclusionList(Container):
+    signed_summary: SignedInclusionListSummary
```

Comment:
can we have slot here or in the signed summary?

Reply:
done!